### PR TITLE
Blobstore2

### PR DESCRIPTION
**Pull request by lilian-hunt**
BlobstoreUploadUrlServlet.java is present in the blobstore base branch but it is still coming up as added. I am not sure why.

### CODE REVIEW

**Review by jupi3200 — COMMENTED** [review]
PRs should be named with more descriptive names 'Blobstore2' doesn't tell the reader what functionality you this PR is adding.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -62,7 +62,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     for (Entity entity : results.asIterable()) {
       String id = KeyFactory.keyToString(entity.getKey());
       Map<String, String> commentAttributes = new HashMap<>();
-      commentAttributes.put("comment", (String) entity.getProperty("comment"));
+      commentAttributes.put("commentText", (String) entity.getProperty("commentText"));
```
Comment (jupi3200) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:65]: Wasn't this part of a previous PR?

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -51,6 +51,7 @@ function getFeedback() {
 
     const commentListElement = document.getElementById('comment-container');
     commentListElement.innerHTML = '';
+    console.log(comments);
```
Comment (jupi3200) [inline portfolio/src/main/webapp/script.js]: Should this be in the PR?

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (lilian-hunt) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:65]: Yes, sorry I'm still trying to figure out exactly what to do with these branches and that's why I hadn't sent these links to the team chat yet. I think something went wrong when creating the branches.

Comment (lilian-hunt) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:65]: I think I will close these pull requests and try the branching again.

**Pull request closed without merging**